src/shared/modules/features/versionedFeatures.js

```diff
@@ -80,6 +80,17 @@ export const getUsedDbName = state => {
   return getActiveDbName(state)
 }
 
+export const getDefaultBoltScheme = serverVersion => {
```

Comment:
```
export const getDefaultBoltScheme = serverVersion =>
  semver.valid(serverVersion) && semver.gte(serverVersion, NEO4J_4_0) ? 'neo4j://' : 'bolt://'
```
Maybe?

Reply:
Maybe. Makes it a bit harder to read, but not too bad. Merging without any changes because it follows the same pattern as other functions in that file.
But a candidate for a future refactor, for sure!

Thanks!